### PR TITLE
Add a query command

### PR DESCRIPTION
**Pull request by necessary129**
NOT TESTED

Parameters: [Roll number|Email|discord user] [which property to query]

Should currently be able to get the name/rollnumber/email.

No testing was done. I'm too lazy to set up a testing env right now. Maybe some time later.

`bonsai` needs to be installed in the env of the bot to be able to query LDAP.

2 new ENV_VARS:
 - `LDAP_HOST`: Self explanatory. should be `ldap://ldap.iiit.ac.in` for IIITH
 - `LDAP_BASE`: Self explanatory. should be `dc=iiit,dc=ac,dc=in` for IIITH

### CODE REVIEW

**Comment by necessary129** [comment]
1. Only query the details of users already `.verify` ied. So the whole LDAP stuff is not needed.
2. Only allow `.query` in servers with `is_academic` flag set.

File: bot/main.py [hunk 1]
```diff
@@ -51,9 +54,7 @@ def get_realname_from_discordid(user_id):
 
 
 async def send_link(ctx):
-    await ctx.send(
-        f"<{PORTAL_LINK}>\nSign in through our portal, and try again."
-    )
+    await ctx.send(f"<{PORTAL_LINK}>\nSign in through our portal, and try again.")
```
Comment (Groverkss) [inline bot/main.py:57]: Unrelated change.

Comment (necessary129) [reply bot/main.py:57]: These are made by `black`. but I'll revert them ig.

File: bot/requirements.txt [hunk 2]
```diff
@@ -16,4 +16,4 @@ python-dotenv==0.17.1
 regex==2021.4.4
 toml==0.10.2
 typing-extensions==3.10.0.0
-yarl==1.6.3
+yarl==1.6.3
```
[the remark this answers is not in the document]
Comment (necessary129) [reply bot/requirements.txt:19]: It was already in `requirements.txt`. I just added a newline. Remove this?

File: bot/main.py (hunk 1 above)
Comment (Groverkss) [reply bot/main.py]: You can do that as a separate patch if you like, but it's still unrelated to this patch.
